src/Appwrite/SMS/Adapter/Dialog360.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Appwrite\SMS\Adapter;
+
+use Appwrite\SMS\Adapter;
+
+// Reference Material
+// https://docs.360dialog.com/docs/whatsapp-api/whatsapp-api/sandbox#php
+
+class Dialog360 extends Adapter
```

Comment:
Would you please implement this in [utopia-php/messaging](https://github.com/utopia-php/messaging) following [these](https://github.com/utopia-php/messaging/blob/main/docs/add-new-adapter.md) instructions?